build.gradle

```diff
@@ -21,36 +14,47 @@ configurations {
 
 allprojects {
   group = "com.linkedin.coral"
+  apply plugin: "com.diffplug.spotless"
+
   repositories {
     mavenCentral()
     maven {
       url 'https://linkedin.bintray.com/maven/'
     }
   }
-}
 
-subprojects {
-  apply plugin: "com.github.hierynomus.license"
-
-  license {
-    header = file("${rootDir}/gradle/license/LICENSE_HEADER")
-    ext.year = Calendar.getInstance().get(Calendar.YEAR)
-    skipExistingHeaders true
-    ignoreFailures false
-    strictCheck true
-
-    exclude '**/*.avsc'
-    exclude '**/*.json'
+  spotless {
+    groovyGradle {
+      target '**/*.gradle'
+      endWithNewline()
+      greclipse().configFile("${rootDir}/gradle/spotless/codestyle-eclipse.xml")
+    }
+    format 'markdown', {
+      target 'CONTRIBUTING.md', 'README.md', 'docs/*.md'
```

Comment:
Is using `**/*.md` not an option here? Just so that any new markdown files are also automatically targeted.

Reply:
Fixed.